CWP/papers/HSF-CWP-2017-05_analysis/latex/cwp-wg-analysis-interpretation.tex

```diff
@@ -11,8 +11,8 @@
 \usepackage[utf8]{inputenc}
 
 \usepackage[style=numeric-comp,sorting=none]{biblatex}
-\addbibresource{../../HSF-CWP-2017-01_roadmap/latex/cwp.bib}
-\addbibresource{../../HSF-CWP-2017-01_roadmap/latex/cwp-chapters.bib}
+\addbibresource{cwp.bib}
```

Comment:
Why did this change from the shared `.bib` file? We should only have one shared resource for reference.